gulp/tasks/watch.js

```diff
@@ -7,7 +7,7 @@ function changed(event) {
 }
 
 gulp.task('watch', ['build'], function () {
-  gulp.watch([ paths.source ], [ 'es6', browserSync.reload ]).on('change', changed);
+  gulp.watch([ paths.source ], [ 'es6', 'lint', browserSync.reload ]).on('change', changed);
```

Comment:
The lint task is just informational. Run 'lint' after browserSync.reload in order to prevent the slow down.